### PR TITLE
Add ntlm support

### PR DESCRIPTION
**Pull request by pgremo**
I have added support for ntlm to the CI's that could support it.  There wasn't any way to plug it into request so it's if branched

### CODE REVIEW

File: app/services/TfsProxy.js [hunk 1]
```diff
@@ -9,21 +10,27 @@ module.exports = function () {
...
+            ntlm.get({
+              'url': url,
+              'username': self.configuration.username,
+              'password': self.configuration.password
+            }, function (error, response) {
+              callback(error, JSON.parse(response.body));
+            });
+          } else {
             request({
-                'url': self.configuration.tfsProxyUrl || tryGetTfsProxyUrlOfDocker(),
```
Comment (marcells) [inline app/services/TfsProxy.js]: Where is this stuff gone?

**Comment by marcells** [comment]
Cool, thanks. :+1:

But, could you please extend the `Readme.md`. I made a comment in the TFS service, too.

Maybe abstracting request/ntlm in a general method would be good, so we can get rid of the if/else-switch in the services.

File: app/services/TfsProxy.js (hunk 1 above)
Comment (pgremo) [reply app/services/TfsProxy.js]: bad copy paste.  it's fixed

**Comment by pgremo** [comment]
I've extracted the makeRequest and updated the readme

**Comment by thinrichs** [comment]
+1 We also need this, what is the status with pulling in this PR?

**Comment by marcells** [comment]
Thanks! Sorry for the late response. ;-)

**Comment by modulexcite** [comment]
Thank you for an excellent tool, any plans on releasing new version now that this PR is merged?

**Comment by pgremo** [comment]
will a new release also be a new release of the docker container?

**Comment by marcells** [comment]
Sorry guys, just forgot to make a new release. ;-) Version 0.9.21 is finished now.
Thanks for all your effort!

https://github.com/marcells/node-build-monitor/releases/tag/v0.9.21
https://hub.docker.com/r/marcells/node-build-monitor/tags/

**Comment by modulexcite** [comment]
Thanks, you're the best.

**Comment by marcells** [comment]
:D

**Pull request merged**